CHANGELOG.md

```diff
@@ -1,23 +1,81 @@
-Changes by Version
-==================
-
 ### A message to people of Russia 🇷🇺
 
 If you currently live in Russia, please read [this message](./_To_People_of_Russia.md).
 
+Changes by Version
+==================
+
+<details>
+<summary>next release template</summary>
+
 next release (yyyy-mm-dd)
 -------------------
 
 ### Backend Changes
 
-#### ⛔ Breaking Changes
+run `make changelog` to generate content
```

Comment:
What is the motivation for adding this? It's already documented in RELEASE.md.

Reply:
It was distracting / confusing having empty sections especially after copying the template, and there were many sections that are no longer necessary.